### DISCUSSION TITLE
Issues with Windows 11 OOBE

### DISCUSSION THREAD

**Discussion by Denisxiang** [utmapp/UTM#3331] [UTM Help (macOS)]
Windows 11 not responding to clicking and keyboard inputs
When I installed Windows 11 ARM and booted into the OOBE, in my tests two to three times, Windows 11 was not responding to mouse clicks or keyboard inputs, it just freezes and had to restart the VM a couple of times, Has anyone experienced this issue? if you have, do let me know

Thanks

Configuration
UTM Version: 2.4.1
OS Version: macOS Monterey 12.0.1
Chip: Apple Silicon M1

(No crash logs as it did not crash UTM, just the VM not responding to any inputs)

**Comment by conath** [comment]
I have read about this on UTM Discord with some users suggesting that the "virtualization=on" flag is causing the problem (using the VHDX method to install Windows 11 ARM). The ISO method seems to not have this problem.
